accounts/package.json

```diff
@@ -47,6 +47,7 @@
     "@rollup/plugin-json": "^4.0.1",
     "@rollup/plugin-replace": "^2.3.0",
     "archiver": "^5.3.0",
+    "chromedriver": "^93.0.1",
```

Comment:
Isn't that one coming from the `tests/acceptance/package.json` ? (the same would apply for settings/package.json)